bigquery_etl/shredder/config.py

```diff
@@ -355,3 +356,26 @@ def find_glean_targets(pool, client, project=SHARED_PROD):
             if any(field.name == CLIENT_ID for field in table.schema)
         },
     }
+
+
+EXPERIMENT_ANALYSIS = "moz-fx-data-experiments"
+
+
+def find_experiment_analysis_targets(pool, client, project=EXPERIMENT_ANALYSIS):
+    """Return a dict like DELETE_TARGETS for experiment analysis tables."""
+    datasets = {dataset.dataset_id for dataset in client.list_datasets(project)}
+
+    tables = [
+        table
+        for tables in pool.map(
+            client.list_tables,
+            [bigquery.DatasetReference(project, dataset_id) for dataset_id in datasets],
+        )
+        for table in tables
+        if table.table_type != "VIEW" and not table.table_id.startswith("statistics_")
+    ]
+
+    return {
+        experiment_analysis_target(qualified_table_id(table)): DESKTOP_SRC
```

Comment:
this should reuse `client_id_target`, instead of defining `experiment_analysis_target` with identical behavior
```suggestion
        client_id_target(table=qualified_table_id(table)): DESKTOP_SRC
```